File: libmvp/src/main/java/com/mr/k/mvp/base/IBaseCallBack.java
package com.mr.k.mvp.base;

import com.mr.k.mvp.exception.ResultException;

/*
 * created by Cherry on 2019-12-20
 **/
public interface IBaseCallBack<T> {

    void onSuccess(T data);

    void onFail(ResultException e);
}

File: libmvpkotlin/src/main/java/com/mr/k/mvp/base/IBaseCallBack.kt
package com.mr.k.mvp.base

import com.mr.k.mvp.exception.ResultException

/*
 * created by Cherry on 2019-12-20
 **/
interface IBaseCallBack<T> {
    fun onSuccess(data: T)
    fun onFail(e: ResultException)
}